src/utils/ldat.ts

```diff
@@ -177,9 +177,9 @@ async function testLDAT() {
 }
 
 function serializeMeta(obj) {
-  var str: string[] = []
-  for (var p in obj) {
-    if (obj.hasOwnProperty(p)) {
+  const str: string[] = []
+  for (const p in obj) {
+    if (Object.prototype.hasOwnProperty.call(obj, p)) {
```

Comment:
@Evanfeenstra eslint was erroring on this let me know if this change looks good, one of the few odd ones

Reply:
in my PR i worked around this with 
```diff
@@ -176,15 +176,13 @@ async function testLDAT() {
   sphinxLogger.info(parseLDAT(token2))
 }
 
-function serializeMeta(obj) {
-  var str: string[] = []
-  for (var p in obj) {
-    if (obj.hasOwnProperty(p)) {
-      str.push(encodeURIComponent(p) + '=' + encodeURIComponent(obj[p]))
-    }
-  }
-  str.sort((a, b) => (a > b ? 1 : -1))
-  return str.join('&')
+function serializeMeta(obj: { [k: string]: any }) {
+  return Object.entries(obj)
+    .map(p => {
+      return `${encodeURIComponent(p[0])}=${encodeURIComponent(p[1])}`
+    })
+    .sort((a, b) => (a > b ? 1 : -1))
+    .join('&')
 }
 
 function deserializeMeta(str) {
```

basically using Object.entries instead of a for loop